.github/workflows/ci.yaml

```diff
@@ -0,0 +1,64 @@
+name: Continuous Integration
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
```

Comment:
Good idea to move from Travis to GitHub Actions. 👍🏻 

I have two points.

1. Here go-version 1.19 is specified, but '1.20' is specified for the `integration` job, I feel like they should both be '1.20' but is it difficult?

2. Also, setup-go supports `go-version-file` input, which reads the `go.mod` file to get the version. Maybe this could be used https://github.com/actions/setup-go/blob/main/action.yml#L7-L8

(the current go.mod is 1.12 so this needs to be updated when using `go-version-file` )
https://github.com/DATA-DOG/go-txdb/blob/164f0b8f537414c9a05123ff55ded733052e38e7/go.mod#L3

Reply:
I have upgrade go-version in lint to 1.20. I think just keep the 1.12 in go.mod and lint with 1.20 is enough. @stefafafan